### PR TITLE
Blog: ZAP on raspberry pi

### PR DESCRIPTION
**Pull request by eingengraou**
Signed-off-by:eingengraou
miracleoct21@gmail.com

**- Summary**

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by kingthorin — CHANGES_REQUESTED** [review]
Maybe also give "site/content/blog/2022-08-25-zap-on-raspberry-pi/images/images (7).jpeg" a better name? (The other screenshots seem well named.)

The blank lines in the markdown could definitely be reduced.

I'll do a more complete review later.

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+title: "Running ZAP on a raspberry pi"
+summary: >
+  Setting up ZAP on the raspberry pi.
+images:
+- https://www.zaproxy.org/blog/2022-08-25-zap-on-raspberry-pi/images/Screenshot_2022-08-20-18-01-55-325_com.realvnc.viewer.android.jpg
+type: post
+tags:
+- blog
+- gsoc
```
Comment (thc202) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:10]: Remove this one.

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+date: "2022-08-25"
+authors: 
+    - eingengraou
```
Comment (psiinon) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:13]: You also need to update [authors.yml](https://github.com/zaproxy/zaproxy-website/blob/master/site/data/authors.yml) and optionally include a picture of yourself or an image you'd like to use for yourself.

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+The single reason was that i had issues with steady power supply and needed something to hack with all the time even if i'm away from home so it would be possible to spend more time on targets.
+
+
+Now that you know the reason behind the whole idea , we can dive right in.
+
+
+
+
+## Getting started 
+ Disclaimer: There's currently no method to run [zap](https://github.com/zaproxy/zaproxy) on an android device alone.
```
Comment (psiinon) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:32]: "ZAP" should always be capitalized, unless it is part of a filename such as `zap.sh`

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+Go to [ZAP official download page](https://www.zaproxy.org/download/) and download the ***`core cross platform package`*** zip file.
+
+![ZAP core zip download screen](images/Screenshot_2022-08-20-12-33-22-028_com.realvnc.viewer.android.jpg)
+
+After the download is completed unzip the downloaded package and you'll have a folder like ***`ZAP_version_number`*** , run the zap.sh script in the folder and ZAP will start.
+
+![ZAP beginning screen](images/Screenshot_2022-08-20-12-57-35-168_com.realvnc.viewer.android.jpg)
+
+The reason for using the core version of ZAP is because it doesn't  contain unneccesary componets like the in-built browser that doesn't  work on the set up.
```
Comment (psiinon) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:134]: typo: "components"

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+![raspberry pi vnc terminal screen](images/Screenshot_2022-08-20-09-17-08-469_com.realvnc.viewer.android.jpg)
+
+## Step 4: Installing ZAP on the raspberry pi
+Before we start ZAP installation, we need to install java8 or higher in our raspberry pi (i used java11 for this installation)
+
+```
+sudo apt-get install openjdk-8-jdk
+```
+
+Go to [ZAP official download page](https://www.zaproxy.org/download/) and download the ***`core cross platform package`*** zip file.
```
Comment (psiinon) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:126]: As its the same site we can replace `https://www.zaproxy.org/download/` with `/download/`

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+Using any ssh client , connect to the ip address(192.168.43.112 in my case) of raspberry pi on the network using the preconfigured ssh credentials in Step 1 to enable access to a terminal window
+   
+![Neofetch ssh screen](images/IMG_20220819_105041.jpg)
+
+Run the following commands to install a VNC server on the raspberry pi
+
+* Update the raspberry pi repository list
+
+```
```
Comment (psiinon) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:78]: We usually use put "bash" after the 3 backquotes - that then uses white on black as per https://www.zaproxy.org/faq/how-can-you-import-post-requests/#proxy-another-tool - but its up to you. Either change none or all 😉

**Review by psiinon — CHANGES_REQUESTED** [review]
Just minor typos and comments

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+After installing the Raspberry PI imager according to your respective operating system. Launch the software with administrative permissions so you can be able to write into the SD card.
+
+![Raspberry pi image default screen](images/download.png)
+
+Then click the ***'CHOOSE OS'*** button , select ***'CUSTOM'*** from the drop down menu that will be displayed , then you can locate the Raspberry pi image you downloaded earlier.
+
+Now plug in your card reader with the micro SD card inserted into any of the various USB ports on your computer and click on the ***'CHOOSE STORAGE'*** button and locate your SD card in the options displayed (usually your sd card is the only option displayed if no other storage devices are plugged into your computer).
+
+The ***'WRITE'*** button at this point would change colour depicting that you're ready to write into your SD card , but before we write into the storage device click the 'SETTINGS' icon below the gear button to access options to pre-configure WiFi , SSH (recommended)  and so on and set according to your preferrences.
```
Comment (thc202) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:57]: `preferences`

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+* Run tightvncserver
+```
+    sudo tightvncserver
+```
+
+Now create a password for the VNC server .There's no need to create a view only password 
+
+The Raspberry Pi has now started a VNC server session at ***your raspberry pi ip:1*** remember this address as it is needed later.
+
+Install any VNC client software from the Google Playstore and launch the application add new host using ***your raspberry pi ip:1*** as Host address, use the user that created the vnc server on your raspberry pi and the password you created.
```
Comment (thc202) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:96]: `Play store`

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md (hunk 7 above)
Comment (kingthorin) [reply site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:57]: Good catch

File: site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+![ZAP core zip download screen](images/Screenshot_2022-08-20-12-33-22-028_com.realvnc.viewer.android.jpg)
+
+After the download is completed unzip the downloaded package and you'll have a folder like ***`ZAP_version_number`*** , run the zap.sh script in the folder and ZAP will start.
+
+![ZAP beginning screen](images/Screenshot_2022-08-20-12-57-35-168_com.realvnc.viewer.android.jpg)
+
+The reason for using the core version of ZAP is because it doesn't  contain unneccesary components like the in-built browser that doesn't  work on the set up.
+
+All you have to do now is install any browser of your choice, install ZAP's CA certificate and [set up the proxy](https://www.zaproxy.org/docs/desktop/start/proxies/) on the browser and you're good to go.
```
Comment (thc202) [inline site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md:136]: This link needs to be changed.

**Review by psiinon — APPROVED** [review]

**Comment by psiinon** [comment]
Doh, no, just me being stupid :P

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]
Yup looks good to me.

**Pull request merged**